### ISSUE TITLE
Proposed documentation on Image Title Field Usage as Alt Text

### ISSUE THREAD

**Issue by ArafatAbdussalam** [wagtail/guide#376]
## **What is Image title field and alt-text?**

The image title field represents the title of an image. However, images also need to have alt-text. Alt-text provides the descriptive meaning of an image and can be read by assistive technologies. 

## **Why should the Image title field usage as alt text be documented?**
As part of the ATAG criteria, certain authors are unable to view images and have to rely on alt-text which is an alternative to non-text content. Documenting the usage of the accessibility feature will enable content authors to be able understand how to change an image alt-text. In certain cases, alt-texts will enable content authors to be able to interact with different images and including the documentation on the user guide means that the authors do not need to find it hard to achieve such task.

### **Step-by-step details**
- The documentation lies within the image documentation guide
- explain how to reach the alt-text field
- explain how to save the alt-text

**Comment by thibaudcolas** [comment]
Now done in https://guide.wagtail.org/en-latest/how-to-guides/manage-images/
